.github/workflows/pull-request-checks.yml

```diff
@@ -20,6 +20,8 @@ jobs:
         uses: actions/checkout@v2
       - name: Swift Lint
         run: swiftlint
+      - name: Update CocoaPods Specs
```

Comment:
To make sure the `pod lib lint` checks are based on the latest https://github.com/CocoaPods/Specs